source/particle/world.cc

```diff
@@ -1317,9 +1319,9 @@ namespace aspect
       {
         prm.enter_subsection("Tracers");
         {
-          prm.declare_entry ("Load balancing strategy", "none",
-                             Patterns::Selection ("none|remove particles|"
-                                                  "remove and add particles|repartition"),
+          prm.declare_entry ("Load balancing strategy", "repartition",
+                             Patterns::MultipleSelection ("none|remove particles|add particles|"
+                                                          "remove and add particles|repartition"),
```

Comment:
In a follow-up patch, would you mind documenting here what the different choices imply? Also, what it means to list more than one of them.